### PR TITLE
(RE-6785) Add huaweios platform

### PR DESCRIPTION
**Pull request by ScottGarman**
Tested by building a repo package and inspecting it - looks sane to me.

### CODE REVIEW

File: configs/platforms/huaweios-6-powerpc.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+platform "huaweios-6-powerpc" do |plat|
+  plat.servicedir "/etc/init.d"
+  plat.defaultdir "/etc/default"
+  # HuaweiOS is based on Debian 8 bug uses sysv instead of systemd
```
Comment (shermdog) [inline configs/platforms/huaweios-6-powerpc.rb]: *But

**Comment by ScottGarman** [comment]
Fixed a spelling typo in a comment.

**Comment by shermdog** [comment]
:+1:

**Comment by shrug** [comment]
Does it wind up in a huaweios repo, or in jessie?

**Comment by ScottGarman** [comment]
The repo url is huaweios specific

**Comment by mckern** [comment]
@ScottGarman have you got a URL example?

**Comment by ScottGarman** [comment]
Yep, here is the contents of /etc/apt/sources.list.d/puppetlabs-pc1.list:

```
# Puppetlabs PC1 huaweios Repository
deb http://apt.puppetlabs.com huaweios PC1

# Puppetlabs PC1 huaweios Source Repository
# The source repos are commented out by default because we
# do not always make sources available for all packages or
# for all platforms. If you want to access the source repos,
# uncomment the following line.
#deb-src http://apt.puppetlabs.com huaweios PC1
```

**Pull request merged**